sentry-spring/src/main/java/io/sentry/spring/tracing/SentryTracingFilter.java

```diff
@@ -67,16 +67,22 @@ protected void doFilterInternal(
       final ITransaction transaction = startTransaction(httpRequest, sentryTraceHeader);
       try {
         filterChain.doFilter(httpRequest, httpResponse);
+      } catch (Exception e) {
+        // exceptions that are not handled by Spring
+        transaction.setStatus(SpanStatus.INTERNAL_ERROR);
```

Comment:
should we `throw e;`?

Reply:
There is no need to setThrowable as the exception is captured while transaction is active - SentryClient sets the span context on the event.